test/CMakeLists.txt

```diff
@@ -36,12 +36,12 @@ pmpo_add_exe(testTracking testTracking.cpp)
 pmpo_add_exe(testRebuild testRebuild.cpp)
 pmpo_add_exe(timeAssmblyWachspress testTiming.cpp)
 
-add_test(NAME unit_test COMMAND ./unitTest)
-add_test(NAME mp_unit_test COMMAND ./mpUnitTest)
-add_test(NAME test_wachspress COMMAND ./testWachspress)
-add_test(NAME test_reconstruction COMMAND ./testReconstruction)
-add_test(NAME test_tracking COMMAND ./testTracking)
-add_test(NAME test_rebuild COMMAND ./testRebuild)
+add_test(NAME unit_test COMMAND mpirun --bind-to core -np 1 ./unitTest)
+add_test(NAME mp_unit_test COMMAND mpirun --bind-to core -np 1 ./mpUnitTest)
+add_test(NAME test_wachspress COMMAND mpirun --bind-to core -np 1 ./testWachspress)
+add_test(NAME test_reconstruction COMMAND mpirun --bind-to core -np 1 ./testReconstruction)
+add_test(NAME test_tracking COMMAND mpirun --bind-to core -np 1 ./testTracking)
+add_test(NAME test_rebuild COMMAND mpirun --bind-to core -np 1 ./testRebuild)
```

Comment:
I'm not sure `--bind-to core` is portable.  `mpirun` should also not be hardcoded here - the user should be passing the name of the binary/script needed to run mpi jobs. 

This applies to the other uses below as well.

Reply:
This is an old problem I mentioned in May: when I want to run the tests on Aimos, if I do:`mpirun --bind-to core -np 1 c1 test --test-dir buildPolyMPO-GPU/` on Aimos, once the first test called finalized, the other tests will just freeze. So I need to call `mpirun` every time.
This evening when I tried to run the tests on Perlmutter, I found that there is no `mpirun` on Perlmutter, so I added a new commit to check if there is `mpirun`: https://github.com/SCOREC/polyMPO/pull/10/commits/994e2b51bd221b87a4379b905c8977ab3a525871
This is just a temporary solution to pass all the tests on all the environments (Cranium, Perlmutter, and Aimos).
I don't know much about the ctest and how to use `mpirun` with it.

Reply:
OK.  We can make this portable in another PR.